package.json

```diff
@@ -7,5 +7,8 @@
   "main": "server.js",
   "dependencies": {
     "express-gateway": "^1.16.10"
+  },
+  "devDependencies": {
```

Comment:
It this just a dev dependency? I see that it is used in server.js. npm install will not install dotenv on production. And when you load server.js on production it will fail.

Reply:
@Abdulkadir98 could you comment on this?

Reply:
Hello @patelvp, I have removed it as a dev dependency and added it as a normal dependency. Can you please have a look